src/main/java/com/vonage/quickstart/verify/StartVerification.java

```diff
@@ -19,26 +19,25 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package com.nexmo.quickstart.verify;
+package com.vonage.quickstart.verify;
 
-import com.nexmo.client.NexmoClient;
-import com.nexmo.client.verify.VerifyResponse;
-import com.nexmo.client.verify.VerifyStatus;
-
-import static com.nexmo.quickstart.Util.configureLogging;
-import static com.nexmo.quickstart.Util.envVar;
+import com.vonage.client.VonageClient;
+import com.vonage.client.verify.VerifyResponse;
+import com.vonage.client.verify.VerifyStatus;
+import static com.vonage.quickstart.Util.envVar;
+import static com.vonage.quickstart.Util.configureLogging;
 
 public class StartVerification {
     public static void main(String[] args) {
         configureLogging();
 
-        String NEXMO_API_KEY = envVar("NEXMO_API_KEY");
-        String NEXMO_API_SECRET = envVar("NEXMO_API_SECRET");
+        String VONAGE_API_KEY = envVar("VONAGE_API_KEY");
+        String VONAGE_API_SECRET = envVar("VONAGE_API_SECRET");
         String RECIPIENT_NUMBER = envVar("RECIPIENT_NUMBER");
 
 
-        NexmoClient client = NexmoClient.builder().apiKey(NEXMO_API_KEY).apiSecret(NEXMO_API_SECRET).build();
-        VerifyResponse response = client.getVerifyClient().verify(RECIPIENT_NUMBER, "NEXMO");
+        VonageClient client = VonageClient.builder().apiKey(VONAGE_API_KEY).apiSecret(VONAGE_API_SECRET).build();
+        VerifyResponse response = client.getVerifyClient().verify(RECIPIENT_NUMBER, "VONAGE");
```

Comment:
Compiler is complaining that this is a deprecated API. Think we should be using the modern one.

Reply:
Good catch. That was one of those changes that didn't go well with the rebase apparently.
fixed